### ISSUE TITLE
support luamb environments

### ISSUE THREAD

**Issue by seanjensengrey** [pkulchenko/ZeroBraneStudio#687]
[Luamb](https://github.com/un-def/luamb) is an environment manager somewhat like `pip` for Python but based off of https://github.com/mpeterv/hererocks

It would be nice to support per project environments, for tab completion and library code navigation.

**Comment by pkulchenko** [comment]
It seems to be a command-line environment, so I'm not sure what king of tab completion may be useful, but in any case someone would have to provide the auto-complete descriptions, similar to how it's done for [Love](https://github.com/pkulchenko/ZeroBraneStudio/blob/master/api/lua/love2d.lua) and other APIs.

**Comment by seanjensengrey** [comment]
Ok, I was/am confused how autocomplete is managed or navigate to source. 

Down in `/Users/seanjensengrey/.luambenvs/test0/share/lua/5.2`

I have

```
... elided
├── mime.lua
├── redis.lua
├── socket
│   ├── ftp.lua
│   ├── headers.lua
│   ├── http.lua
│   ├── smtp.lua
│   ├── tp.lua
│   └── url.lua
└── socket.lua
```

Is there a way to support passing that lib path to the IDE so that navigate to source works?

**Comment by pkulchenko** [comment]
No, it doesn't (yet) work this way. The IDE expects a description as documented [here](https://studio.zerobrane.com/doc-api-auto-complete).
